src/gsy_e/gsy_e_core/simulation/time_manager.py

```diff
@@ -192,6 +192,18 @@ def handle_slowdown_and_realtime_scm(
 
         self.slot_time_counter = int(time())
 
+    @staticmethod
+    def get_start_time(config: "SimulationConfig") -> DateTime:
+        """Return the start tim of the simulation."""
+        if gsy_e.constants.RUN_IN_REALTIME:
+            today = pendulum.today(tz=TIME_ZONE)
+            seconds_since_midnight = time() - today.int_timestamp
+            slot_no = int(seconds_since_midnight // config.slot_length.seconds) + 1
+            start_time = config.start_date + duration(seconds=slot_no*config.slot_length.seconds)
```

Comment:
I am fairly certain that we discussed this, however why not adding support for the SCM days of delay for Canary Networks? There is a chance that it is already included in the `config.start_date` however I will see it later on the PR if so.

Reply:
The problem is, that here we are in the init of the simulation, where the "slot resumed" is not calculated yet. Here we like to buffer the profiles and only want to start at 12:00 on the day the CN was started then and not on the config.start_date, that could be e.g. 3 month in the past.

Reply:
Got it, I just checked the callers of this method and indeed this is not called elsewhere so all good. Only thing I suggest though is to name this `get_start_time_on_init` instead, in order to avoid confusion when trying to use this method in the future. What do you think?

Reply:
Agreed, let me change this.